doc/devices/ahs_local.rst

```diff
@@ -0,0 +1,105 @@
+The local AHS device
+====================
+
+The local AHS device of the PennyLane-Braket plugin runs Analogue Hamiltonian Simulation on the local Braket SDK. This
+could be either utilizing the processors of your own PC, or those of a `Braket notebook instance <https://docs.aws.amazon.com/braket/latest/developerguide/braket-get-started-create-notebook.html>`_ hosted on AWS.
+
+This device is useful for small-scale simulations in which the time of sending a job to a remote service would add
+an unnecessary overhead. It can also be used for rapid prototyping before running a computation
+on a paid-for remote service.
+
+
+Usage
+~~~~~
+
+After the Braket SDK and the plugin are installed you immediately have access to the `local Braket AHS device <https://docs.aws.amazon.com/braket/latest/developerguide/braket-devices.html#braket-simulator-ahs-local>`_ in PennyLane.
```

Comment:
The link is pointing to the "local AHS simulator", rather than a device (in the PennyLane nomenclature), so I'd suggest to say
```suggestion
After the Braket SDK and the plugin are installed you immediately have access to the `local Braket AHS simulator <https://docs.aws.amazon.com/braket/latest/developerguide/braket-devices.html#braket-simulator-ahs-local>`_ in PennyLane.
```